multus_sriov_cni.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash -x
+
+export RECLONE=${RECLONE:-true}
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+export PATH=/usr/local/go/bin/:/opt/cni/bin/:/home_stack/cni/cni/cnitool/$PATH
```

Comment:
It should be `:$PATH` also add `kubernetes/third_party/etcd` it is need when run kubernetes